3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.iOS/active-directory-xamarin-intune.iOS.csproj

```diff
@@ -52,6 +52,7 @@
     <MtouchLink>None</MtouchLink>
     <MtouchInterpreter>-all</MtouchInterpreter>
     <BuildIpa>true</BuildIpa>
+    <MtouchNoSymbolStrip>true</MtouchNoSymbolStrip>

```

Comment:
This setting can be removed

Reply:
That is kept. So developer can upload it to the Intune Portal.

Reply:
This setting shouldn't impact developer's ability to upload an app to the Intune portal. By default, native debug symbols are stripped from the app binary in Xamarin apps. External developers won'y have the ability to step through the native ObjC code within the Intune bindings anyway so there's no benefit to enabling this setting for them.

Reply:
@SameerK-MSFT - Not sure if you saw my reply above. You can disable this setting in Visual Studio's iOS Build settings:

Right click the active-directory-xamarin.iOS project.
Select "Options", 
Select "iOS Build" in the left pane.
Enable the "Strip native debugging symbols" option.